libjolie/src/jolie/lang/parse/SemanticVerifier.java

```diff
@@ -386,6 +387,9 @@ public void validate()
 		}
 	}
 
+	@Override
+	public void visit( ReThrowStatement n ) {}
```

Comment:
Which also could be too restrictive. Consider the following example (not checked):
`define X
{
       if ( global.insideFaultHandler ) {
       	  rethrow
       }
}

main
{
     install( default => global.insideFaultHandler = true; X );
     throw( f ) 
}`